### ISSUE TITLE
v1.3.0 missing object expanders on web interface

### ISSUE THREAD

**Issue by HyperSprite** [getsentry/sentry-electron#219]
- [x] Review the documentation: https://docs.sentry.io/platforms/javascript/electron/
- [x] Search for existing issues: https://github.com/getsentry/sentry-electron/issues
- [x] Use the latest release: https://github.com/getsentry/sentry-electron/releases
- [ ] Provide a link to the affected event from your Sentry account (I can DM a link if necessary but would rather not post one here)

## Versions + Platform

- [x] SKD version - `@sentry/electron: 1.3.0,`
- [x] Electron version - `electron: 7.1.12`
- [x] Platform - `macOS`

## Description

We have a working version of our app running with v1.2.0 and our Extra data objects have `+` expanders in them that can be expanded to show more information. 

![Screen Shot 2020-03-17 at 5 59 48 PM](https://user-images.githubusercontent.com/9909591/76915370-5e8fc280-687a-11ea-996e-4396da5810a2.png)

When I upgrade to v1.3.0 these `+` expanders are replaced with [Object] and can not be expanded.

![Screen Shot 2020-03-17 at 5 58 46 PM](https://user-images.githubusercontent.com/9909591/76915374-62234980-687a-11ea-8393-3d2a754a784f.png)

If I go back to v1.2.0, the expanders return

**Comment by timfish** [comment]
v1.2.1 introduced [some changes](https://github.com/getsentry/sentry-electron/pull/215/files) to prepare for Electron 8 where IPC uses Chrome structured clone algorithm.

if you download the full JSON from Sentry, are the objects there?

![image](https://user-images.githubusercontent.com/1150298/76963089-5ede9b00-6918-11ea-92ef-25ca5ddd4511.png)

**Comment by HyperSprite** [comment]
The JSON has the same [Object]

```json
"_persist": {
  "rehydrated":true,
  "version":27
},
"accounts": {
  "5711175e-e9c4-5a35-85c9-4fcd9edf504d": "[Object]",
  "42469d28-8b23-5ac4-974b-af9631a4f5b9":"[Object]",
},
"apps": {},
```

**Comment by timfish** [comment]
@HyperSprite can you confirm that your objects are also like this for v1.2.1 where this change was made?

**Comment by HyperSprite** [comment]
On v1.2.1 I can see the expanders like we had in v1.2.0
![Screen Shot 2020-03-18 at 9 52 40 AM](https://user-images.githubusercontent.com/9909591/76985993-49f50e00-68fe-11ea-9324-fd6954afcd73.png)
![Screen Shot 2020-03-18 at 9 52 50 AM](https://user-images.githubusercontent.com/9909591/76986001-4e212b80-68fe-11ea-969c-1006084627ec.png)

**Comment by timfish** [comment]
Thanks for checking this. Then it looks like this difference was caused by all the Sentry [dependencies getting bumped](https://github.com/getsentry/sentry-electron/commit/7703de16471edf8efcd07269985a60835b5eda54). Maybe @HazAT can shed some light on what could have changed?

**Comment by timfish** [comment]
We have just released v3.0.0 of the SDK which includes a lot of changes and improvements since this issue was opened. 

If you can still reproduce this with v3, please open a new issue and I'll look into it.
